drracket/drracket/private/tooltip.rkt

```diff
@@ -87,7 +87,7 @@
       (send parent reflow-container))
 
     ;; private field for gc reasons
-    (define font-size-callback (λ (p v) (set! tooltip-font #f) (update-size)))
+    (define (font-size-callback p v) (set! tooltip-font #f) (update-size))
```

Comment:
This is another situation where I wanted to emphasize that this is per-object storage (so we keep the pref callback alive as long as the object is alive but they can both be gc'd afterwards).